libs/application/templates/financial-statements-inao/src/lib/utils/helpers.ts

```diff
@@ -0,0 +1,66 @@
+import { getValueViaPath } from '@island.is/application/core'
+import { ExternalData, FormValue } from '@island.is/application/types'
+import subYears from 'date-fns/subYears'
+import getYear from 'date-fns/getYear'
+import { TOTAL, USERTYPE, YES } from '../constants'
+
+export const getTotal = (values: Record<string, string>, key: string) => {
+  if (!values[key]) {
+    return 0
+  }
+  const total = Object.entries(values[key])
+    .filter(([k, v]) => k !== TOTAL && !isNaN(Number(v)))
+    .map((foo) => {
+      return foo
+    })
+    .map(([_k, v]) => Number(v))
+    .reduce((prev, current) => {
+      return (prev += current)
+    }, 0)
+  return total
+}
+
+export const formatNumber = (num: number) => num.toLocaleString('de-DE')
```

Comment:
could probably use the react-intl package to do this (and other stuff in this file)... at least something to check out